docs/manuals/source/Appendix/DisasterRecoveryUsingBareos.rst

```diff
@@ -128,19 +158,339 @@ to create the rescue image. If you used the configuration example above, you wil
+   Using Catalog "MyCatalog"
+             jobid: 113
+               job: RestoreFiles.2024-06-28_14.25.22_15
+              name: RestoreFiles
+       purgedfiles: 0
+              type: R
+             level: F
+          clientid: 4
+            client: bareosclient-fd
+         jobstatus: T
+         schedtime: 2024-06-28 14:25:22
+         starttime: 2024-06-28 14:25:24
+           endtime: 2024-06-28 14:25:51
+       realendtime: 2024-06-28 14:25:51
+          duration: 00:00:27
+          jobtdate: 1,719,584,751
+      volsessionid: 1
+    volsessiontime: 1,719,583,621
+          jobfiles: 63,889
+          jobbytes: 2,716,580,999
+         joberrors: 0
+   jobmissingfiles: 0
+            poolid: 0
+          poolname: 
+        priorjobid: 0
+         filesetid: 0
+           fileset: 
+   Restored 2.7G
+   Restore job finished successfully.
+   Bareos restore finished.
+   Created SELinux /mnt/local/.autorelabel file : after reboot SELinux will relabel all files
+   Recreating directories (with permissions) from /var/lib/rear/recovery/directories_permissions_owner_group
+   [...]
+   Updated initramfs with new drivers for this system.
+   Installing GRUB2 boot loader...
+   Determining where to install GRUB2 (no GRUB2_INSTALL_DEVICES specified)
+   Found possible boot disk /dev/vda - installing GRUB2 there
+   Finished 'recover'. The target system is mounted at '/mnt/local'.
+   Exiting rear recover (PID 499) and its descendant processes ...
+   Running exit tasks
+   RESCUE bareosclient:~ # 
+
+The restored system can be found under the :file:`/mnt/local` directory.
+After restoring the files, ReaR restored the bootloader.
+The recovery is now completed and the system can be rebooted.
+
+The ReaR Bareos backend also works in ReaRs **Automatic Recover** mode,
+selectable at boot,
+which recreates the system without manual interaction.
+
+
+Advanced ReaR Bareos Backend Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _section-rear-snapshot-package-installation:
+
+Installing a ReaR Snapshot Package
+''''''''''''''''''''''''''''''''''
+
+The current (2024) stable release of ReaR is 2.7,
+which does not support Bareos > 22.
+Therefore until a newer version is released,
+a ReaR snapshot package must be used:
+
+* Download the ZIP file that best matches your distribution from https://github.com/rear/rear/releases/tag/snapshot
+* Unpack the ZIP file and install the package on the client.
+
+For details, refer to https://relax-and-recover.org/download/
+
+
+.. _section-named-console-for-rear:
+
+Configure a Named Console for ReaR
+''''''''''''''''''''''''''''''''''
+
+The default :command:`bconsole` configuration on the |dir| offers full access to the |dir|.
+This is normally not wanted from a client systems.
+
+Instead you configure a :config:option:`dir/console` for every Bareos client using ReaR,
+limiting the access to only one system and the required console commands:
+
+.. code-block:: bareosconfig
```

Comment:
This feels very cumbersome if you have lots of clients... If there is a way to make this more simple and still secure then maybe would be good to mention it.

Reply:
There are two ways we address this:

1. when #1875 get merged, it still require one console per client, but using a rear profile, in can be reduced to just a few settings
2. we like to add a mode, where the rescue image only contain the bareos-fd, not the bconsole. This got the limitation, that the restore must be triggered from the server, but simplifies the client a lot. Also using bconsole on the server is the typical use-case.

Reply:
Once this is done, we can extend the documentation to reflect that.